src/Components/SearchGhost.js

```diff
@@ -0,0 +1,30 @@
+import Box from "@mui/material/Box";
+import Container from "@mui/material/Container";
+import Skeleton from "@mui/material/Skeleton";
+
+export default function SearchGhost() {
+  return (
+    <Container maxWidth="sm" style={{ paddingLeft: "16px" }}>
+      <GhostListItem />
```

Comment:
I like these ghosts and I think they look pretty good.  I think we could maybe try with having fewer -- generally the loading 'ghosts' page doesn't have to line up 1:1 with the rendered page, and this way I think the screen would be less overwhelming, but the user's thought process wouldn't be as interrupted during load repaints as it would be with a loading spinner.  Apparently, UX research says that users perceive ghost pages load faster than spinners (b/c there is more to look at and start parsing while the loading happens, I think) and their user journey feels less interrupted during loading.  Maybe just use 5 here?

Reply:
Woah - that's really interesting. Sounds good to me!